frontend/src/components/PlayerTile.js

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components/macro';
+import { Gi3DMeeple } from 'react-icons/all';
+import { IconContext } from 'react-icons';
+
+export default function PlayerTile({ existingPlayer, onChange }) {
+    return (
+        <Wrapper>
+            <IconContext.Provider
```

Comment:
🤯 Didn't know about `IconContext`, interesting!

Reply:
the usage is here was the easier way to pass down the color information. for some reason it did not work when I select it the common way...